README.md

```diff
@@ -93,13 +93,31 @@ Type: `function`
 
 Filter out files. [more details](#hooks-options)
 
+**Use-case**:  Omit [dll-chunks] ([issue](https://github.com/danethurber/webpack-manifest-plugin/issues/46)):
+```js
+  filter: ({chunk, file}) => {
+    return chunk.isInitial();
+  }
+```
+You can generate additional manifest-file with a dll-chunks only using the [DllPlugin].
+
+[dll-chunks]: https://webpack.js.org/guides/code-splitting/#dynamic-imports
+[DllPlugin]: https://webpack.js.org/plugins/dll-plugin/
 
 ### `options.map`
 
 Type: `function`
 
 Modify files details before the manifest is created. [more details](#hooks-options)
 
+**Use-case**: Add hash information to manifest file for [SRI] ([issue](https://github.com/danethurber/webpack-manifest-plugin/issues/35), [issue](https://github.com/danethurber/webpack-manifest-plugin/issues/55)):
+```js
+  map: ({chunk, file}) => ({ file, hash: chunk.source.integrity }),
+  generate: (seed, files) => files.reduce((manifest, {name, path}) => manifest), seed),
```

Comment:
I don't think that is working?

Reply:
I'm not sure too 😂
I just rewrited from `reduce` to `generate`

after:
```
map: ({chunk, file}) => {
  return {
    file,
    hash: chunk.source.integrity
  };
}

reduce: (manifest, {chunk, file}) => {
  return manifest;
}
```

default generate is:
```
`(seed, files) => files.reduce((manifest, {name, path}) => ({...manifest, [name]: path}), seed)`
```